workspaces/src/types.rs

```diff
@@ -11,6 +11,10 @@ use near_primitives::logging::pretty_hash;
 use near_primitives::serialize::{from_base, to_base};
 use serde::{Deserialize, Serialize};
 
+/// Nonce for transactions.
+pub type Nonce = u64;
+
+/// Gas is a type for storing amount of gas.
```

Comment:
```suggestion
/// Gas units that are used to execute operations for a transaction.
```

I know this comment came from primitives, but it's not great. Feel free to re-word, but this is my stab at a better comment